percy/percy.test.js

```diff
@@ -0,0 +1,247 @@
+  )
+  await percySnapshotWithWait(transitive, 'Itinerary (with transitive)', true)
+
+  const routes = await loadPath('/route')
+  await percySnapshotWithWait(routes, 'Route Viewer (with transitive)', true)
+})
+*/
+
+test('OTP-RR', async () => {
+  const page = await loadPath('/')
+  await page.setViewport({
+    height: 1080,
+    width: 1920
+  })
+
+  await percySnapshotWithWait(page, 'Main Page (without styling)')
+
+  // Plan a trip
+  await page.goto(
+    `http://localhost:${MOCK_SERVER_PORT}/#/?ui_activeSearch=5rzujqghc&ui_activeItinerary=0&fromPlace=Opus Music Store%2C Decatur%2C GA%3A%3A33.77505%2C-84.300178&toPlace=Five Points Station (MARTA Stop ID 908981)%3A%3A33.753837%2C-84.391397&date=2022-03-09&time=09%3A58&arriveBy=false&mode=WALK%2CBUS%2CSUBWAY%2CTRAM%2CFLEX_EGRESS%2CFLEX_ACCESS%2CFLEX_DIRECT&showIntermediateStops=true&maxWalkDistance=1207&optimize=QUICK&walkSpeed=1.34&ignoreRealtimeUpdates=true&wheelchair=false&numItineraries=3&otherThanPreferredRoutesPenalty=900`
+  )
+  await page.waitForNavigation({ waitUntil: 'networkidle2' })
+  await page.waitForSelector('.title')
+
+  await percySnapshotWithWait(page, 'Batch Itinerary')
+
+  // Select a trip
+  await page.waitForSelector('.title:nth-of-type(1)')
+  await page.click('.title:nth-of-type(1)')
+
+  await percySnapshotWithWait(page, 'Batch Itinerary Selected')
+
+  // Open Trip Viewer
+  await page.waitForTimeout(2000)
+  const [tripViewerButton] = await page.$x(
+    "//button[contains(., 'Trip Viewer')]"
+  )
+  await tripViewerButton.click()
+  await page.waitForSelector('div.trip-viewer-body')
+  await page.waitForTimeout(1000)
+  await percySnapshotWithWait(page, 'Trip Viewer')
+
+  // Open stop viewer from trip viewer
+  await page.click(
+    'div.trip-viewer-body > div:nth-child(3) > div.stop-button-container > button'
+  )
+  await page.waitForSelector('.stop-viewer')
+
+  await percySnapshotWithWait(page, 'Stop Viewer')
+
+  // Open schedule view
+  await page.waitForSelector('button.link-button.pull-right')
+  await page.click('button.link-button.pull-right')
+  await page.waitForTimeout(3000) // Slow animation
+  await percySnapshotWithWait(page, 'Schedule Viewer')
+
+  // Open route viewer
+  const [routeViewerButton] = await page.$x(
+    "//button[contains(., 'View Routes')]"
+  )
+  await routeViewerButton.click()
+  await page.waitForSelector('.route-viewer')
+  await page.waitForTimeout(5000)
+
+  await percySnapshotWithWait(page, 'Route Viewer')
+
+  // Open Specific Route`
+  const [busRouteButton] = await page.$x(
+    "//span[contains(., 'Sugarloaf Mills - Lindbergh Center')]"
+  )
+  await busRouteButton.click()
+  await page.waitForSelector('#headsign-selector')
+
+  await percySnapshotWithWait(page, 'Route Viewer Showing Route 410')
+
+  // View multiple patterns
+  // Click second option
+  const sugarloafOption = await page.$$eval(
```

Comment:
Maybe rename to `firstPattern` and the other variable to `secondPattern`

Reply:
That does feel better thanks! Done in 67e1dc094ba9047bede64664b66a399085257cd0